src/style.css

```diff
@@ -1,3 +1,73 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
+* {
```

Comment:
Using the universal selector '*' impacts performance negatively. It's better to apply styles to specific elements or classes that require them.